docs/gcve-private-cloud-on-foundation-4-1-0.md

```diff
@@ -0,0 +1,246 @@
+# How to customize Foundation v4.1.0 for Google Cloud VMware Engine (GCVE) Private Cloud deployment
```

Comment:
can we place this modules in root level folder ? where all common modules are kept